master/internal/grpcutil/api.go

```diff
@@ -26,22 +26,25 @@ const jsonPretty = "application/json+pretty"
 
 // NewGRPCServer creates a Determined gRPC service.
 func NewGRPCServer(db *db.PgDB, srv proto.DeterminedServer) *grpc.Server {
-	logger := logrus.NewEntry(logrus.StandardLogger())
+	logger := logrus.New()
```

Comment:
I spent some time reading the logrus code - afaict it seems totally fine to have two logrus loggers but I'm always happy to be surprised..

Reply:
Yep, that is my reading as well -- assuming the underlying `io.Writer` is safe to write to concurrently, which I think should be the case here with `os.Stderr`.